main.js

```diff
@@ -0,0 +1,85 @@
+// the game board to display with the 1st peg being the "start" peg
+let board = [
+  [5,4,3,2,1],
+  [],
+  []
+]
+
+// function to execute to make the board visible
+function displayBoard() {
+  console.clear()
+  board.map(function(pegs) {
+    console.log(`--- ${pegs}`)
+    return
```

Comment:
for each wont require you to return.
Returning something empty, its a sign something is off